src/extension.ts

```diff
@@ -25,7 +26,8 @@ import MongoDBLanguageClient from './mongo/languageClient';
 import { Reporter } from './telemetry';
 import { UserCancelledError } from './errors';
 import { DocDBCommands } from './docdb/commands';
-import { DialogBoxResponses } from './constants'
+import { DialogBoxResponses } from './constants';
+import { error } from 'util';
```

Comment:
Why do you import error? I don't think that type exists?

Reply:
It seems to be a VSCode auto-completion thing. If I've used error as a variable anywhere and later renamed it, it imports the symbol it can't find in the file. 
I noticed it then, but didn't figure out the cause.